controller/app/pending_ops_models/remove_alias_op.rb

```diff
@@ -31,6 +33,8 @@ def rollback
       application.save!
     end
 
+    OpenShift::SsoService.register_gear(gear) unless gear.removed
```

Comment:
Why the inconsistency in execute vs rollback? Please add a comment why SsoService.register_alias is not a provided option. In any case, we should add comment in SsoService class to indicate that register_gear is expected to be idempotent.